<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sample_4_1Class</class>
 <widget class="QWidget" name="sample_4_1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>字符串的输入与输出</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelNum">
       <property name="text">
        <string>数量：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="EditNum">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelSingalPrice">
       <property name="text">
        <string>单价：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="EditSinglePrice">
       <property name="text">
        <string>7.2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnCal">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>计算总价</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelGlobalPrice">
       <property name="text">
        <string>总价：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="EditGlobalPrice"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelTenHex">
       <property name="text">
        <string>十进制：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="EditTenHex">
       <property name="text">
        <string>15</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnTenHex">
       <property name="text">
        <string>转换成其他进制</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelTwoHex">
       <property name="text">
        <string>二进制：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="EditTwoHex">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnTwoHex">
       <property name="text">
        <string>转换成其他进制</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelHex">
       <property name="text">
        <string>十六进制：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="EditHex">
       <property name="text">
        <string>F</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnHex">
       <property name="text">
        <string>转换成其他进制</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sample_4_1.qrc"/>
 </resources>
 <connections/>
</ui>
